source/type/pb_converter.cpp

```diff
@@ -133,7 +133,7 @@ struct PbCreator::Filler::PtObjVisitor: public boost::static_visitor<> {
                                                            impacted_stop->mutable_amended_stop_time());
 
             // we need to get the base stoptime
-            const auto* base_st = impact.aux_info.get_base_stop_time(stu);
+            const auto* base_st = stu.stop_time.get_base_stop_time();
```

Comment:
So the only "possible regression" is here (Kraken used to care about index here), but it looks to me that it is used when displaying a disruption... To be confirmed if possible (poke @TeXitoi ?).

Then it wouldn't be too bad to display a lollipop disruption badly.

Reply:
I really don't know if that's critical, but in theory, sherbrooke has lolipop and only delays, thus we break something (that might not be used).

A cheap lolipop fix can be added, something like "1. checking that we are the ith stop time using this stop point, and then searching for the ith stop time using this stop point in the base".